images/tkg/docs/examples/adding_os_pkg_repos.md

```diff
@@ -0,0 +1,106 @@
+# Adding new OS packages and configuring the repositories or sources
+
+## Use case
+
+As a user I want to add new OS packages example `pkg1` and `pkg2` and also I want to configure ubuntu/photon sources/repositories to an internal mirror to build the node images in an air-gapped scenario. Configuring new sources/repositories is also useful when we want to install internally built software.
```

Comment:
As a user I want to 
1. add new OS packages such as `pkg1` and `pkg2` 
2. configure sources/repositories to an internal mirror to build the node images in an air-gapped scenario. 
Configuring new sources/repositories is also useful when we want to install internally built software.